chai/chai.d.ts

```diff
@@ -249,10 +249,7 @@ declare module Chai {
         Throw(fn: Function, errType: Function, msg?: string): void;
         Throw(fn: Function, errType: Function, regExp: RegExp): void;
 
-        doesNotThrow(fn: Function, msg?: string): void;
-        doesNotThrow(fn: Function, regExp: RegExp): void;
-        doesNotThrow(fn: Function, errType: Function, msg?: string): void;
-        doesNotThrow(fn: Function, errType: Function, regExp: RegExp): void;
+        doesNotThrow(fn: Function, err?: Function | RegExp, msg?: string): void;
```

Comment:
I don't feel comfortable naming an argument `err` and allowing it to be either an error or a regular expression. Also, this doesn't account for the scenario in line 255 where there is a function, an error and then a regex.